Solutions/GoogleCloudPlatformDNS/Analytic Rules/GCPDNSMultipleErrorsFromIp.yaml

```diff
@@ -0,0 +1,36 @@
+id: 7e81a935-5e91-45a5-92fd-3b58c180513b
+name: Google DNS - Multiple errors for source
+description: |
+  'Detects multiple errors for the same source IP address.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: GCPDNSDataConnector
+    dataTypes:
+      - imDNSGCPCloudDNS
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CommandAndControl
+relevantTechniques:
+  - T1095
+query: |
+  let threshold = 10;
+  let err = dynamic(['NXDOMAIN', 'SERVFAIL', 'REFUSED']);
+  imDNSGCPCloudDNS
+  | where EventResultDetail in~ (err)

```

Comment:
Since you are using the parser in queries, you will have to change field name EventResultDetail to EventResultDetails.
Only then the validations would pass. Thank You.

Reply:
fixed